scripts/test_duplicate_values.sh

```diff
@@ -9,13 +9,13 @@ REPO_DIR="$(dirname "$SCRIPT_DIR")"
 # Check values in files
 cd $REPO_DIR
 SW_VERSION="$(grep 'appVersion\s=' service-worker.js | sed -E "s/[^[:digit:]]+([^\"']+).*/\1/")"
-APP_VERSION="$(grep 'params\[.appVersion' www/js/app.js | sed -E "s/[^[:digit:]]+([^\"']+).*/\1/")"
+APP_VERSION="$(grep 'params\[.appVersion' www/js/init.js | sed -E "s/[^[:digit:]]+([^\"']+).*/\1/")"
 echo "service-worker.js : $SW_VERSION"
-echo "app.js            : $APP_VERSION"
+echo "init.js            : $APP_VERSION"
```

Comment:
You need to remove one space after `init.js` so that the colons align in the display.